cmd/rukpakctl/cmd/version.go

```diff
@@ -0,0 +1,24 @@
+/*
+Copyright © 2022 NAME HERE <EMAIL ADDRESS>
```

Comment:
Is there a copyright header that rukpak uses? This seems like an improper copyright header and the year is wrong

Reply:
Yeah, this is a copy/paste. I'll fix this one. Probably need to do a scrub for the others.